README.md

```diff
@@ -94,6 +94,12 @@ this you can use the `--include-open` option:
 
     $ ./vendor/bin/changelog-generator generate --user=doctrine --repository=migrations --milestone=2.0 --include-open
 
+## Including Date after the Milestone
+
+For a good changelog your like show the release date of each version. For this you can use the `--include-date` option:
+
+    $ ./vendor/bin/changelog-generator generate --user=doctrine --repository=migrations --milestone=2.0 --include-open
```

Comment:
The option used is wrong here. Used `--include-open` instead of `--include-date` in the example command.